internal/authtoken/options.go

```diff
@@ -1,5 +1,16 @@
 package authtoken
 
+import (
+	"time"
+
+	"github.com/hashicorp/boundary/internal/db"
+)
+
+var (
+	defaultTokenTimeToLiveDuration  = 7 * 24 * time.Hour
+	defaultTokenTimeToStaleDuration = 7 * time.Hour
```

Comment:
Note that you have changed this value from 24 hours to 7 hours. That's quite a drastic change. Do we actually want this?

Reply:
Good catch, fixed